setup.cfg

```
@@ -0,0 +1,22 @@
+[metadata]
+name = zppy-e3sm-chem-diags
+version = 1.0.0
+author = Hsiang-He Lee
+author_email = lee1061@llnl.gov
+description = Chemistry Diagnostics Package
+long_description = file: README.md
+long_description_content_type = text/x-md
+url = https://github.com/E3SM-Project/ChemDyg
+project_urls =
+    Bug Tracker = https://github.com/E3SM-Project/ChemDyg/issues
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+
+[options]
+packages = find:
+include_package_data = True
+python_requires = >=3.8
+install_requires =
+   zppy >=2.2.0
```

We will need to increase this constraint once there is a release of `zppy` that supports plugins.

We should add more dependencies here.  This is one of the reasons I would like to pull out the python files, rather than embedding them in the bash files.  It would be much easier to determine what dependencies are required if they were always at the top of the python files.

---

I believe these are all your dependencies.  Please add version constraints if you know them.  This is especially important if you don't think your code is compatible with the latest versions of these packages.  Sometimes you will only learn this when it's too late (and it's pretty inconvenient).  So we can update these as we need to.
```suggestion
   cartopy
   matplotlib
   netcdf4
   numpy
   pandas
   scipy
   xarray
   zppy >=2.2.0
```

---

I went ahead and committed these changes.